deploy/mapppacker.tf

```diff
@@ -0,0 +1,97 @@
+# Stuff managed outside of this module
+data "azurerm_resource_group" "this" {
+  name = var.resource_group_name
+}
+data "azurerm_container_registry" "this" {
+  name                = var.container_registry_name
+  resource_group_name = var.resource_group_name
+}
+data "azurerm_storage_account" "this" {
+  name                = var.storage_account_name
+  resource_group_name = var.resource_group_name
+}
+
+# Azure Storage Queue
+resource "azurerm_storage_queue" "mappacker_requests" {
+  name                 = "mappacker-requests"
+  storage_account_name = var.storage_account_name
+}
+
+# Azure Container App
+resource "azurerm_container_app_environment" "this" {
+  name                = var.container_app_env_name
+  resource_group_name = var.resource_group_name
+  location            = var.container_app_environment_location
+}
+resource "azurerm_container_app" "mapgl_tile_renderer" {
+  name                         = var.container_app_name
+  resource_group_name          = var.resource_group_name
+  container_app_environment_id = azurerm_container_app_environment.this.id
+  revision_mode                = "Single"
+
+  registry {
+    server               = data.azurerm_container_registry.this.login_server
+    username             = data.azurerm_container_registry.this.admin_username
+    password_secret_name = "containerregistry"
+  }
+  # This secret is not used by the application code, but is used by for
+  # Azure Container App to download the image.
+  # TODO: Remove this secret by instead using a user-assigned managed identity
+  # for Container App to connect to registry (see: registry {identity:...})
+  secret {
+    name  = "containerregistry"
+    value = data.azurerm_container_registry.this.admin_password
+  }
+
+  # This secret is used for scale rules.
+  secret {
+    name  = "queueconnection"
+    value = data.azurerm_storage_account.this.primary_connection_string
+  }
+
+  template {
+    container {
+      name   = "mbgl-tile-renderer"
+      image  = "${data.azurerm_container_registry.this.login_server}/mbgl-tile-renderer:test-3"
+      cpu    = "0.25"
+      memory = "0.5Gi"
```

Comment:
btw this is a dinky server.  Not sure if it can handle a large request.  Very easy to scale up though.